web/gui/dashboard_info.js

```diff
@@ -579,6 +579,36 @@ netdataDashboard.menu = {
         info: undefined
     },
 
+    'iis': {
+        title: 'IIS',
+        icon: '<i class="fas fa-eye"></i>',
+        info: undefined
+    },
+
+    'mssql': {
+        title: 'MS SQL Server',
+        icon: '<i class="fas fa-database"></i>',
+        info: undefined
+    },
+
+    'ad': {
+        title: 'Active Directory (AD)',
```

Comment:
Do we really need `(AD)` here?

Reply:
I added it because the others have only `AD`. I did not want menus with large names like `Active Directory Certification Service`. This is an indication to simplify user understanding. At least we see a lot references like this in papers.

Reply:
I think it is so well known, that all people know what AD (in IT) stands for.

Reply:
Last time I was working with AD (yes, it was the first years), this was not so used, people was still learning and moving from NT 4 to it, I agree that nowadays this is more common than I was expecting.

Reply:
NT 4 is 1996. Been a while!

Reply:
But, Active directory came with Windows 2000.